codecs/codecs-btp/src/main/java/org/interledger/codecs/btp/AsnBtpPacketCodec.java

```diff
@@ -71,7 +76,7 @@ public void encode(T value) {
 
   protected void onRequestIdChanged(long code) {
 
-    BtpMessageType type = BtpMessageType.fromCode(getValueAt(0));
+    BtpMessageType type = fromCode(getValueAt(0));
```

Comment:
Minor nit: I prefer to keep method imports class-qualified, like this: `BtpMessageType.fromCode(getValueAt(0));` so it's overt where this is coming from. It's probably not a big deal in this case, but I have had Intellij auto-qualify static methods and if there's ever ambiguity it sometimes picks the wrong one (and it's easier to miss if the actual class is statically imported at the top of the file).

The one exception to this style rule is in unit-test assertions, where I think readability outweighs things.